pkg/core/blockchain.go

```diff
@@ -2240,7 +2240,8 @@ unsubloop:
 // CalculateClaimable calculates the amount of GAS generated by owning specified
 // amount of NEO between specified blocks.
 func (bc *Blockchain) CalculateClaimable(acc util.Uint160, endHeight uint32) (*big.Int, error) {
-	return bc.contracts.NEO.CalculateBonus(bc.dao, acc, endHeight)
+	ic := bc.newInteropContext(trigger.Application, bc.dao, &block.Block{Header: block.Header{Index: bc.BlockHeight() + 1}}, nil)
```

Comment:
Let's use `bc.getFakeNextBlock(...)` for it.

Reply:
Done.